talent/flexible_holidays_working_schedule_remote_work.md

```diff
@@ -8,10 +8,14 @@ At source{d} you will have 23 days of holidays to use however you wish. This mea
 
 Unlike in some other companies we don't count the weekends of bank holidays that fall inside of your holidays. This means that if you take two weeks off it will count as 10 holidays (2x Monday-Friday). This is the norm in most countries but some companies in some countries also count the weekend in between as part of your holidays.
 
-You can also work during bank holidays or weekends and exchange those days for additional holidays. You just have to let your team know that you are going to do so. Again, try to do it with time so that everyone can plan accordingly.
+You can also work during bank holidays or weekends and exchange those days for additional holidays. You just have to let your team know that you are going to do so. Again, try to do it with time so that everyone can plan accordingly.The company will include an extra day of vacations in the annual total amount.
+
+If by the company’s necessities an employee has to work during a festive day or a weekend, he will receive an extra day of vacations to the day of work resulting in the addition of two vacation days added to the total annual amount of holidays. 
```

Comment:
Does this mean 1 day worked on a holiday/weekend = 2 days added to the total holidays? AFAIK people have been getting just 1 for the day they had to work.

Reply:
I don't think so, when people work because the Company need it, people ask me to add 2 extra days. If they ask only for one it's because they wanted to work that day (or maybe, for your comment, because they didn't know it)

Reply:
Thanks, I really didn't know that! Did we have this anywhere before?
I guess I miss a few days then heh, will check that out at some point. ;)

Reply:
@marnovo this is when explicitly we ask someone to work during the weekend.